Azure/azure-cli
# [SQL] `az sql midb move/copy`: Add destination subscription Id for managed database move/copy

**Related command**


- az sql midb move start
- az sql midb move complete
- az sql midb move cancel
- 
- az sql midb copy start
- az sql midb copy complete
- az sql midb copy cancel

**Description**

Before this cross subscription db move/copy was not possible because we could not know the subscription of the target managed instance. In order to fix this I am adding optional parameter **--dest-subscription-id**.

**Testing Guide**


This example has DestinationSubscriptionId so we assume that is different from the SourceSubscriptionId
az sql midb copy complete --name mydb --resource-group MyResourceGroup --managed-instance MyInstance --dest-mi DestinationInstance --dest-rg DestinationResourceGroup --dest-subscription-id DestinationSubscriptionId

In this example we leave it out so the DestinationSubscriptionId will be the same as the SourceSubscriptionId
az sql midb copy complete --name mydb --resource-group MyResourceGroup --managed-instance MyInstance --dest-mi DestinationInstance --dest-rg DestinationResourceGroup



---

This checklist is used to make sure that common guidelines for a pull request are followed.
